packages/core/src/tools/data.js

```diff
@@ -153,7 +153,7 @@ const flattenPaths = (data, { preserve = {} } = {}) => {
 
 // A simpler, and memory-friendlier version of _.truncate()
 const simpleTruncate = (string, length, suffix) => {
-  if (string === undefined) {
+  if (string == null) {
```

Comment:
this now returns for `null` and `undefined`; before we were converting `null` to `''`, which was weird.